javascript-and-json/main.js

```diff
@@ -0,0 +1,33 @@
+var booksArray = [
+  {
+    isbn: '978-1-60309-388-0',
+    title: 'The Jekyll Island Chronicles (Book One): A Machine Age War',
+    author: 'Steve Nedvidek'
+  },
+  {
+    isbn: '978-1-60309-426-9',
+    title: "The Jekyll Island Chronicles (Book Two): A Devil's Reach",
+    author: 'Steve Nedvidek'
+  },
+  {
+    isbn: '978-1-60309-493-1',
+    title: 'The Jekyll Island Chronicles (Book Three): A Last Call',
+    author: 'Steve Nedvidek'
+  }
+];
+
+console.log('typeof booksArray:', typeof booksArray);
+console.log('JSON.stringify:', JSON.stringify(booksArray));
+console.log('typeof JSON.stringify:', typeof JSON.stringify(booksArray));
+
+var string = {
```

Comment:
This is not a JSON string. The big thing to remember is that you need single quotes on the outside of the string, and double quotes for all property names and values